lib/Entity/Entity.js

```diff
@@ -25,6 +25,7 @@ class Entity {
         this._url = null;
         this._createMethod = null; // manually set the HTTP-method for create operation, defaults to post
         this._updateMethod = null; // manually set the HTTP-method for update operation, defaults to put
+        this._getMethod = null; // manually set the HTTP-method for the get operation, defaults to get
```

Comment:
Call it `_retrieveMethod` instead, for better compatibility with the CRUD acronym. By the way, it's probably time to add `_deleteMethod`, too.